Library/Homebrew/test/test_cmd_testbot.rb

```diff
@@ -53,7 +53,7 @@ def teardown
   def stub_test_instance
     stub(
       :category => "stub",
-      :log_root => Pathname.pwd,
+      :log_root => Pathname.pwd
```

Comment:
Definitely Ruby 1.8 compatibility. That's the reason why we actively enforce the style without comma [in our RuboCop configuration](https://github.com/Homebrew/brew/blob/db76a0f4cc3838658919570b3453edbcb9ed2fcd/Library/.rubocop.yml#L133-L134), though there's no comment to that end (unlike for many other style rules).